### DISCUSSION TITLE
Is there a way to highlight lines in a text_area?

### DISCUSSION THREAD

**Discussion by hopezh** [marimo-team/marimo#3286] [Q&A]
Is there a way to customize `text_area()` to highlight selected lines, e.g. in boldface or with a background color? 

One such example is the [line-highlight for output](https://github.com/shafayetShafee/line-highlight?tab=readme-ov-file#highlighting-output-line-numbers) in `quarto`:

![line-highlight03](https://github.com/user-attachments/assets/989d3bc3-44a2-4768-84c8-82e057c30f0d)

**Comment by mscolnick** [comment]
@hopezh - you can use `mo.ui.code_editor()` which has syntax highlighting for the language you pass
